templates/projects.html

```diff
@@ -11,14 +11,14 @@
 
 {% block head %}
 <style>
-h1 {
-  margin: 15px 0 30px 0;
+body {
+  padding-left: 20px;
```

Comment:
Is this body side padding meant to be on all pages now? It should go into a single place like base.html.

Reply:
Moved.